src/hooks/useCharacterCount/use-character-count.mdx

```diff
@@ -0,0 +1,39 @@
+import { ArgTypes, Meta, Canvas } from "@storybook/addon-docs";
+
+import * as useCharacterCountStories from "./use-character-count.stories";
+
+<Meta title="Documentation/Hooks/useCharacterCount"/>
+
+# useCharacterCount
+
+`useCharacterCount` is a custom React hook that provides a character counter for input fields. 
```

Comment:
```suggestion
`useCharacterCount` is a custom React hook that provides a character counter for text based input fields. 
```

non-blocking fyi

Reply:
Might also be worth adding that this is not supported with TextEditor atm (again non-blocking)